### PR TITLE
Beam Rifle - From 10 to 5 shots per charge

### PR DESCRIPTION
**Pull request by kriskog**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Skoglol
balance: Reduced the beam rifle shots per charge from 10 to 5.
fix: Beam rifle no longer looks empty before it is empty.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Laser guns got 12 shots, energy guns got 10 lasers, beam rifles for some reason got 10 shots. I feel this is not in line with how powerful it is, especially against blobs. A reduction to 5 shots per charge could hopefully set a requirement for more infrastructure like rechargers to fight a blob efficiently with it, without making them useless.

### CODE REVIEW

**Comment by plapatin** [comment]
prepare the popcorn

**Comment by kriskog** [comment]
Also found a small bug where it applied the empty icon with a shot still left. If this doesn't get merged, I will separate that into its own PR.

**Comment by WarRage333** [comment]
Halve the resource cost too then

This shitty gamemode got so much stuff nerfed just because its so god damn awful

**Comment by Eskjjlj** [comment]
In every gamemode the battles will feel underwhelming for the outside viewer because for one side to win they must have the overwhelming advantage. In this case either the crew steamrolls the blob when it spawns or the blob snowballs without anyone doing much damage.
It would be really difficult to reach a balance where every blob fight feels "clutch" without imposing arbitrary limitations on either side like the "no meta rush" rule CM has.

**Comment by Nickkoranda** [comment]
I don't know if we should be taking rules from CM based on how... senseless they've been. Seeing as blob is a complete domination instead of a greentext I'm biased towards an easier crew victory.

**Comment by GuyonBroadway** [comment]
Please stop trying to change the rest of the game just because you died as blob.

Either git güd or buff blob.

**Comment by lolman360** [comment]
another beam rifle blob salt PR. When will they end?

**Comment by imsxz** [comment]
t. people who havent played as or against blob.

just because you've experienced the strength of an interaction first hand doesn't discredit an argument, it's childish to brush off balance changes as "i ded" unless it's obvious. beam rifles are an absolute "fuck you"/one and done type of thing against blobs the same way grenade lances are vs wizards. There's not much of anything a blob can do about it besides spawning next to either the sec fabs or ore silo to prevent their creation, or pray to god that nobody else has printed one already/ever plans on making one.

plenty of other stuff already hard counters blob, albeit not to this extreme of a level, but enough to take down blobs with some more effort, such as engiborgs, mining kinetic accelerators, emitters, mechas, excessive amounts of laser guns from cargo, and a wide variety of other things. beam guns arent the only way to kill blob, saying they're needed is an over-exaggeration,  they're just extremely overkill and overtuned to an absurd level.

tl;dr there is no gitting gud against PA guns, but there's certainly gitting gud without their current power. even with this change they'll still be a meta choice for killing blobs, just toned down a bit.

**Comment by GuyonBroadway** [comment]
If you need to _"tone down"_ so many things just to make blob a viable gamemode, consider that its blob that needs rebalancing and not everything else in the game. 

Because until some saltlord blob died to them there was never an issue with plasma cutters / flashbangs / xray lasers /etc... and I think its really stupid that the game is being balanced around a random event that hardly ever turns up.


Also if you're a blob and you don't take out the ore slio or R&D to deny the crew weapons they can use against you then more fool you, git gud.

**Comment by kriskog** [comment]
They aren't only strong against blobs though. It is a two shot to crit aoe weapon with zoom that for some reason has 10 shots before needing a charge.

**Comment by kriskog** [comment]
A thing can be too strong without frequently being used.

I can however tell you I used them to great effect from bolted command room on shuttle after solo decimating a blob one time. 50 shots can do a hell of a lot with zoom and reflective walls.

**Comment by GuyonBroadway** [comment]
The shuttle cockpit has a recharger rending your nerf null and pointless in that particular regard.

**Comment by kevinz000** [comment]
as the maker of the things :+1: tbh. 
they were meant to be snipers and I don't take offense at snipers requiring more skill than "spam it blindly".
people start getting killed by these things when the users start learning how to use them to hit a moving target rather than a stationary blob.

**Comment by HideAndSeekLOGIC** [comment]
They're almost impossible to aim and unless the opponent is AFK or blob they are useless. -1.

**Review by Cyberboss — APPROVED** [review]
